fortran/src/hdf5_fortrandll.def.in

```diff
@@ -13,45 +13,62 @@ H5LIB_mp_H5GMTIME
 H5A_mp_H5AWRITE_CHAR_SCALAR
 H5A_mp_H5AREAD_CHAR_SCALAR
 H5A_mp_H5ACREATE_F
+H5A_mp_H5ACREATE_ASYNC_F
 H5A_mp_H5AOPEN_NAME_F
-H5A_mp_H5AOPEN_IDX_F
+@H5_NO_DEPRECATED_SYMBOLS@H5A_mp_H5AOPEN_IDX_F
```

Comment:
Accidental copy/paste or does this control whether the symbol is available?

Reply:
It controls whether the symbol is available.